conf/core/DncExport.conf

```diff
@@ -0,0 +1,7 @@
+{
+  "#" : "Add an OGR option for DNC export",
+  "#" : "DNC uses some MIN INT values to signify NULL's If we don't change this, it throws an",
+  "#" : "exception and dies.",
+  "ogr.strict.checking" : "off",
+  "#" : "end"
+}
```

Comment:
I was thinking we could add the entire list of translation related params to the conf file:
```
translation.script
translation.direction
```
and the assorted `convert.ops+=` as well.

Then the service code only refs a conf file where possible and then also some format (shp/gdb vs osm) related conf files so that we try not to explicitly list config options in the services code if possible.

Reply:
I'll give that a go.

Reply:
It kind of worked. I got the TranslationOp && the geometry mod Op but the output is screwed up.
I'm thinking that this change (All in a conf file) should wait until Brandons PR is merged (https://github.com/ngageoint/hootenanny/pull/3246)

Reply:
Sounds good.  I'm pushing an attempted cuke test fix.